cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTask.java

```diff
@@ -42,11 +43,16 @@
   private static final String EXECUTION_ID = "executionId";
   private static final String TYPE = "type";
   private static final String STATE = "state";
-  private static final String PROPOSAL = "proposal";
+  private static final String TOPIC_PARTITION = "topicPartition";
```

Comment:
Thanks for pointing out the compatibility issue.
the original reason i want to get rid of proposal here is because the 1-to-N mapping between the proposal to intra-broker replica movement task.